simple_history/models.py

```diff
@@ -55,10 +67,13 @@ def __init__(
         history_id_field=None,
         history_change_reason_field=None,
         user_model=None,
-        get_user=default_get_user,
+        get_user=_default_get_user,
         cascade_delete_history=False,
         custom_model_name=None,
         app=None,
+        history_user_id_field=None,
```

Comment:
What is the type of this arg? A classname?

Reply:
Field instance. I.e models.IntegerField(null=True)